### PR TITLE
Avatar badge

### PR DESCRIPTION
**Pull request by masraniJay**
Added Avatar & Badge
Running super late, so no responsiveness or finishing..

### CODE REVIEW

File: components/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    width: fit-content;
+    position: relative;
+    color: var(--primary-color);
+    /* outline: red solid 1px; */
+}
+
+.badge{
+    position: absolute;
+    font-size: small;
+    padding: 0.3rem;
```
Comment (sakshi006) [inline components/badge/badge.css:15]: `padding: 0.3rem 0.5rem`
this would give an even shape to the badge

File: components/badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    width: 1.2rem;
```
Comment (sakshi006) [inline components/badge/badge.css:16]: You can remove the width attribute, padding takes care of the badge text being centered and and badge being circular

File: components/badge/badge.html [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+                                <span class="badge">11</span>
+                            </div>
+                          </div>
+                        </code>
+                    </pre>
+                </div>
+            </div>
+
+            <div class="section">
+                <div class="badge-base"> Badge on an icon
```
Comment (sakshi006) [inline components/badge/badge.html:70]: I would advice you to give an additional class to badges and then fix their positions accordingly. Like, for image, badge is not located on the image, its positioning is not right.

**Review by sakshi006 — CHANGES_REQUESTED** [review]
Suggested some changes.

**Review by arulk8 — APPROVED** [review]

**Pull request merged**